### PR TITLE
Ern-navigation and ern-container-publisher-maven backwardcompatibility support for prev ern versions

### PR DESCRIPTION
**Pull request by Karthiccc23**
- Ern-navigation and ern-container-publisher-maven uses latest version for any ERN version which might break for old versions with new updates.
- This PR adds version control in manifest for above components to support backward compatibility.

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -1489,6 +1489,8 @@
       "appcenter-analytics@3.0.1",
       "appcenter-crashes@3.0.1",
       "appcenter@3.0.1",
+      "ern-navigation@1.2.4",
+      "ern-container-publisher-maven@1.0.5",
```
Comment (friederbluemle) [inline manifest.json:1493]: Those two new entries are still not in alphanumerical order..

**Review by friederbluemle — COMMENTED** [review]
Thank you :+1:

Please confirm if the plan is to release new patches of the ERN platform for versions 0.45, 0.46, 0.47, 0.48, and 0.49? I assume yes, because in those older versions, the added manifest entry for `ern-container-publisher-maven` would have no effect at the moment.

**Comment by Karthiccc23** [comment]
> Thank you 👍
> 
> Please confirm if the plan is to release new patches of the ERN platform for versions 0.45, 0.46, 0.47, 0.48, and 0.49? I assume yes, because in those older versions, the added manifest entry for `ern-container-publisher-maven` would have no effect at the moment.

yes the plan was to add it old versions

**Review by friederbluemle — APPROVED** [review]
:+1:

**Pull request merged**